### PR TITLE
Display Error message for string input

### PR DESCRIPTION
**Pull request by pavithransamy10**
#### Description of Change
Added If else statement in main function so when user inputs anything other than integer error message is shown.I also added a cout method to tell users to enter a Number/Integer.

#### Checklist

- [x ] Added description of change
- [x] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1031"></a>

### CODE REVIEW

**Comment by kvedala** [comment]
code formatting got significantly altered and it does not conform to the repo standards. Please enable "Github Actions" on your fork. No other files or changes to the repo settings are required.
Thank you

**Comment by pavithransamy10** [comment]
> code formatting got significantly altered and it does not conform to the repo standards. Please enable "Github Actions" on your fork. No other files or changes to the repo settings are required.
> Thank you
> ![]

Done Enabling

File: math/sqrt_double.cpp [hunk 1]
```diff
@@ -9,41 +9,54 @@
...
+		return 1 / Sqrt(1 / a);
+	}
+	double l = 0, r = a;
+	/* Epsilon is the precision.
+	A great precision is
+	between 1e-7 and 1e-12.
+	double epsilon = 1e-12;
+	*/
+	double epsilon = 1e-12;
+	while (l <= r) {
```
Comment (kvedala) [inline math/sqrt_double.cpp:30]: Now that I am looking at the code, the correct way seems to be the following:
```c
while ((r - l) > (2.f * epsilon)) { // till the interval is greater than twice the limit
    if (std::fabs(a - mid * mid) < epsilon) {
        return mid;
    }

    double mid = (l + r) / 2.f;  // interval midpoint
    if (mid * mid > a) {
        r = mid;
    }
    else {
        l = mid;
    }
}
```

File: math/sqrt_double.cpp [hunk 2]
```diff
@@ -9,41 +9,54 @@
...
+		assert(n >= 0);
+		// Change this line for a better precision
+		std::cout.precision(12);
+		std::cout << std::fixed << Sqrt(n);
+	}
+	//If user enters a string an error message will be shown
+	else
+	{
+		cout << "please input an integar value";
+	}
```
Comment (kvedala) [inline math/sqrt_double.cpp:61]:
```suggestion
	}
	
	return 0;
```

File: math/sqrt_double.cpp (hunk 1 above)
Comment (kvedala) [reply math/sqrt_double.cpp:30]: Also, in C++11, there is a better way of accessing machine epsilon: [reference](https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon)

**Pull request closed without merging**